nimbus/run.sh

```diff
@@ -1,19 +1,42 @@
 #!/usr/bin/env bash
 
+# Cleanup nimbus directories if they already exist.
+rm -rf /home/user/data/${NODE}
+
 # Refer: https://nimbus.guide/keys.html
 # Running a nimbus VC involves two steps which need to run in order:
 # 1. Importing the validator keys
 # 2. And then actually running the VC
+tmpkeys="/home/validator_keys/tmpkeys"
+mkdir -p ${tmpkeys}
+
 for f in /home/validator_keys/keystore-*.json; do
   echo "Importing key ${f}"
```

Comment:
can we keep this after import? like `echo "Imported key ${f}"`

Reply:
We can but for consistency's sake (see lighthouse/run.sh), let's keep it this way